src/services/likes-service.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
+const LIKES_API = "http://localhost:8080/likes";
```

Comment:
Rather than doing individual ENV variables for each base route, I think it makes more sense to just have an ENV for the plain api url (`http://localhost:8080`), as that will be the only thing that changes between local and prod and then just include the specific base route for the service. For example: `await axios.get('${API_URL}/likes/liked-albums/${userId}');`

This will save us the trouble of managing a different env variable for every service.